### DISCUSSION TITLE
How can I customize the server error page?

### DISCUSSION THREAD

**Discussion by porper** [nuxt/nuxt#9271] [Questions (Nuxt 2)]
### Versions

- nuxt: v2.14.12
- node: v10.16.3

### Reproduction

<details open>

Server error page often appears, like this.
![server-error](https://user-images.githubusercontent.com/30428517/109606890-420b9980-7b62-11eb-9d38-cbbfe2be078f.jpg)
 I want to know the reason, why server error appears, and how can I avoid this page? Is it the client to change the configuration, or the server to optimize the interface?Thanks!

<br>

</details>

**Comment by danielroe** [comment]
@porper You can customise the server error page following the info [in this blog post](https://blog.lichter.io/posts/nuxtjs-change-server-error-page/).
